File: core/src/org/rexellentgames/dungeon/debug/HealCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.entity.creature.player.Player;

public class HealCommand extends ConsoleCommand {
	{
		name = "/heal";
		shortName = "/hl";
		description = "heals you";
	}

	@Override
	public void run(Console console, String[] args) {
		Player.instance.modifyHp(Player.instance.getHpMax() - Player.instance.getHp());
	}
}

File: core/src/org/rexellentgames/dungeon/debug/HealCommand.kt
package org.rexcellentgames.burningknight.debug

import org.rexcellentgames.burningknight.entity.creature.player.Player

class HealCommand : ConsoleCommand("/heal", "/hl", "Heals you") {
    override fun run(console: Console, args: Array<String>) {
        Player.instance.modifyHp(Player.instance.hpMax - Player.instance.hp, null)
    }
}